src/components/eip3643/user/TransferToken.tsx

```diff
@@ -53,6 +55,11 @@ export default function TransferToken({
     tokenSelected?.address as `0x${string}`,
   );
 
+  const { hederaAccountIdError, hederaEVMAccount } = useAccountId(
+    form.values,
+    "toAddress",
```

Comment:
can we pass the value right away? it may simplify things even more, func gets only 1 param and no need to deal with `FormValues` inside the hook, wdyt